cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,281 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.3.0 <http://docs.cask.co/cdap/3.3.0/index.html>`__
+=============================================================
+
+New Features
+------------
+- `CDAP-4280 <https://issues.cask.co/browse/CDAP-4280>`__ -
+  Added the ability to add metadata to artifacts.
+
+- `CDAP-4289 <https://issues.cask.co/browse/CDAP-4289>`__ -
+  Added RESTful APIs to set and get properties for an artifact.
+
+- `CDAP-4228 <https://issues.cask.co/browse/CDAP-4228>`__ -
+  Added the ability to configure which artifact a Hydrator plugin should use.
+
+- `CDAP-4400 <https://issues.cask.co/browse/CDAP-4400>`__ -
+  Added experimental support for batch ETL pipelines run on Spark.
+
+- `CDAP-3945 <https://issues.cask.co/browse/CDAP-3945>`__ -
+  Added a way to limit the number of concurrent schedule runs.
+
+- `CDAP-4043 <https://issues.cask.co/browse/CDAP-4043>`__ -
+  Support for ETL Transforms in Python
+
+- `CDAP-3944 <https://issues.cask.co/browse/CDAP-3944>`__ -
+  PartitionConsumer has support for concurrency, as well as a limit and filter on read.
+
+- `CDAP-4016 <https://issues.cask.co/browse/CDAP-4016>`__ -
+  Added Java-8 support for Script Transform
+
+- `CDAP-4270 <https://issues.cask.co/browse/CDAP-4270>`__ -
+  Added CLI support for metadata and lineage.
+
+- `CDAP-961 <https://issues.cask.co/browse/CDAP-961>`__ -
+  Support on demand (dynamic) dataset instantiation through program runtime context.
+
+- `CDAP-4135 <https://issues.cask.co/browse/CDAP-4135>`__ -
+  Added the capability for MapReduce and Spark programs to localize additional resources during setup.
+
+- `CDAP-4264 <https://issues.cask.co/browse/CDAP-4264>`__ -
+  Added support for automatically annotating CDAP entities with system metadata when they are created or updated.
+
+- `CDAP-3514 <https://issues.cask.co/browse/CDAP-3514>`__ -
+  Added an endpoint to get a count of active queries: /v3/namespaces/<namespace-id>/data/explore/queries/count
+
+- `CDAP-4263 <https://issues.cask.co/browse/CDAP-4263>`__ -
+  Added support for Metadata to Stream's View.
```

Comment:
I think we should update the JIRA also. If this rst can be automatically generated, then we can simply edit the JIRA and rerun the tool?